pkg/diff/diff_test.go

```diff
@@ -48,50 +48,52 @@ func (*testDiffSuite) TestGenerateSQLs(c *C) {
-	c.Assert(deleteSQL, Equals, "DELETE FROM `test`.`atest` WHERE `id` = 1 AND `name` is NULL AND `birthday` = '2018-01-01 00:00:00' AND `update_time` = '10:10:10' AND `money` = 11.1111;")
+	replaceSQL = generateDML("replace", rowsData, tableInfo, "diff_test")
+	deleteSQL = generateDML("delete", rowsData, tableInfo, "diff_test")
+	c.Assert(replaceSQL, Equals, "REPLACE INTO `diff_test`.`atest`(`id`,`name`,`birthday`,`update_time`,`money`) VALUES (1,NULL,'2018-01-01 00:00:00','10:10:10',11.1111);")
+	c.Assert(deleteSQL, Equals, "DELETE FROM `diff_test`.`atest` WHERE `id` = 1 AND `name` is NULL AND `birthday` = '2018-01-01 00:00:00' AND `update_time` = '10:10:10' AND `money` = 11.1111;")
 
 	rowsData["id"] = &dbutil.ColumnData{Data: []byte(""), IsNull: true}
-	replaceSQL = generateDML("replace", rowsData, tableInfo, "test")
-	deleteSQL = generateDML("delete", rowsData, tableInfo, "test")
-	c.Assert(replaceSQL, Equals, "REPLACE INTO `test`.`atest`(`id`,`name`,`birthday`,`update_time`,`money`) VALUES (NULL,NULL,'2018-01-01 00:00:00','10:10:10',11.1111);")
-	c.Assert(deleteSQL, Equals, "DELETE FROM `test`.`atest` WHERE `id` is NULL AND `name` is NULL AND `birthday` = '2018-01-01 00:00:00' AND `update_time` = '10:10:10' AND `money` = 11.1111;")
+	replaceSQL = generateDML("replace", rowsData, tableInfo, "diff_test")
+	deleteSQL = generateDML("delete", rowsData, tableInfo, "diff_test")
+	c.Assert(replaceSQL, Equals, "REPLACE INTO `diff_test`.`atest`(`id`,`name`,`birthday`,`update_time`,`money`) VALUES (NULL,NULL,'2018-01-01 00:00:00','10:10:10',11.1111);")
+	c.Assert(deleteSQL, Equals, "DELETE FROM `diff_test`.`atest` WHERE `id` is NULL AND `name` is NULL AND `birthday` = '2018-01-01 00:00:00' AND `update_time` = '10:10:10' AND `money` = 11.1111;")
 
 	// test value with "'"
 	rowsData["name"] = &dbutil.ColumnData{Data: []byte("a'a"), IsNull: false}
-	replaceSQL = generateDML("replace", rowsData, tableInfo, "test")
-	deleteSQL = generateDML("delete", rowsData, tableInfo, "test")
-	c.Assert(replaceSQL, Equals, "REPLACE INTO `test`.`atest`(`id`,`name`,`birthday`,`update_time`,`money`) VALUES (NULL,'a\\'a','2018-01-01 00:00:00','10:10:10',11.1111);")
-	c.Assert(deleteSQL, Equals, "DELETE FROM `test`.`atest` WHERE `id` is NULL AND `name` = 'a\\'a' AND `birthday` = '2018-01-01 00:00:00' AND `update_time` = '10:10:10' AND `money` = 11.1111;")
+	replaceSQL = generateDML("replace", rowsData, tableInfo, "diff_test")
+	deleteSQL = generateDML("delete", rowsData, tableInfo, "diff_test")
+	c.Assert(replaceSQL, Equals, "REPLACE INTO `diff_test`.`atest`(`id`,`name`,`birthday`,`update_time`,`money`) VALUES (NULL,'a\\'a','2018-01-01 00:00:00','10:10:10',11.1111);")
+	c.Assert(deleteSQL, Equals, "DELETE FROM `diff_test`.`atest` WHERE `id` is NULL AND `name` = 'a\\'a' AND `birthday` = '2018-01-01 00:00:00' AND `update_time` = '10:10:10' AND `money` = 11.1111;")
 }
 
-func (t *testDiffSuite) testDiff(c *C) {
+func (t *testDiffSuite) TestDiff(c *C) {
```

Comment:
this function is not run before 🤢

Reply:
<img width="816" alt="Screen Shot 2020-06-18 at 3 31 50 PM" src="https://user-images.githubusercontent.com/1681864/84991246-d8d12b00-b178-11ea-8dd7-166af83b8bb4.png">

suggest use golangci-lint to help improve the code

Reply:
👍

Reply:
I will update this in next pr